pkg/operation/common/utils.go

```diff
@@ -390,6 +391,10 @@ func DeleteVpa(ctx context.Context, c client.Client, namespace string, isShoot b
 			&corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Name: "vpa-recommender", Namespace: namespace}},
 			&corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Name: "vpa-updater", Namespace: namespace}},
 		)
+
+		resources = append(resources,
+			&admissionregistrationv1beta1.MutatingWebhookConfiguration{ObjectMeta: metav1.ObjectMeta{Name: "vpa-webhook-config"}},
```

Comment:
Move it up to line 342/343.

Reply:
the resources here are deleted using seed client. and if we are deleting VPA for a shoot, then we wouldn't want to delete this resource in the seed.

Reply:
I see, then move it 392/393 instead of having another `append` call :)